examples/braket_features/Simulating_Noise_On_Amazon_Braket.ipynb

```diff
@@ -40,10 +40,10 @@
     "\n",
```

Comment:
<blockquote>you can  apply a quantum channel corresponding to the different gate and noise operations.  </blockquote><p>It looks like there is an extra space after 'can'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Not sure why you're seeing an extra space after "can", but I don't in my own Jupyter notebook</p>

Reply:
I don't see the extra space when looking at the source. Makes me think it could just be a ReviewNB artifact.

Reply:
I agree with Cedric, it seems to be an artifact/weird bug in ReviewNB

Reply:
Whoops, sorry - go ahead then. Thanks a lot!

Reply:
<p>You're welcome!</p>